src/exo/API_scheduling.py

```diff
@@ -1648,6 +1887,14 @@ def divide_loop(proc, loop_cursor, div_const, new_iters, tail="guard", perfect=F
                           to assert that you know the remainder will always
                           be zero (i.e. there is no tail).  You will get an
                           error if the compiler cannot verify this fact itself.
+        rc              - (bool) whether to return relevant cursors
+
+    relevant cursors:
+        outer_loop      - a cursor to the new outer loop
+        inner_loop      - a cursor to the new inner loop
+        ?main_guard     - a cursor to the guard inside the main loop (only for "guard")
+        ?tail_guard     - a cursor to the guard enclosing the tail loop (only for "cut_and_guard")
+        ?tail           - a cursor to the tail loop (only for "cut" and "cut_and_guard")
```

Comment:
Can tail and tail-guard just be collapsed?  Likewise, do we need main-guard, or can one just use the body of the inner-loop?